### PR TITLE
fix(helm): introduce some defaults for db connections and fix python-runner reference

### PR DESCRIPTION
**Pull request by HknLof**
@flippingbits @ChrisRousey Found some errors on a fresh install. Once, tested with new feature, I will update the `minikube-any-namespace.yaml`.

### CODE REVIEW

File: helm-charts/datacater/values.yaml [hunk 1]
```diff
@@ -19,9 +19,9 @@ datacater:
   pythonrunner:
     name: "python-runner"
     image:
-      name: "datacater/python-runner"
+      repository: "datacater/python-runner"
       pullPolicy: IfNotPresent
-      version: "nightly-20230105"
+      version: "nightly"
```
Comment (flippingbits) [inline helm-charts/datacater/values.yaml:24]: We use `nightly` as our `latest`-tag. Is the update of the version to `nightly` intentional despite using a specific nightly build for [other components](https://github.com/DataCater/datacater/blob/main/helm-charts/datacater/values.yaml#L14)?

**Review by flippingbits — COMMENTED** [review]
I have one open question :)

File: helm-charts/datacater/values.yaml (hunk 1 above)
Comment (HknLof) [reply helm-charts/datacater/values.yaml:24]: Will fix it. The reason was, that minikube could not find that tag. Slipped through :)
It is available, though https://hub.docker.com/r/datacater/python-runner/tags?page=1&name=nightly-20230105.

**Review by flippingbits — APPROVED** [review]
Thank you! :)

**Pull request merged**